roles/network_plugin/canal/templates/network.json

```diff
@@ -0,0 +1 @@
+{ "Network": "{{ kube_pods_subnet }}", "SubnetLen": {{ kube_network_node_prefix }}, "Backend": { "Type": "{{ flannel_backend_type }}" } }
```

Comment:
Will fix the quotes, as for defaults - those are required params for roles and defaults come from `inventory/group_vars/all.yml`